website/docs/components/rich-tooltip/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Rich Tooltip
+description: The Rich Tooltip displays structured informational content that is non-essential to the user completing a task.
```

Comment:
For the description, I would probably make this more similar to the `Tooltip` description while also explaining that it supports structured content. (So that the similarities and differences are clearer.)

Reply:
@jorytindall